src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -1763,6 +1772,19 @@ private Task EnumFromStorageFolderAsync(string path, BaseStorageFolder? rootFold
 			}, cancellationToken);
 		}
 
+		private void CheckForSolutionFile()
+		{
+			for (int i = 0; i < filesAndFolders.Count; i++)
+			{
+				if (FileExtensionHelpers.HasExtension(filesAndFolders[i].FileExtension, ".sln"))
+				{
+					SolutionFilePath = filesAndFolders[i].ItemPath;
+					return;
+				}
+			}
+			SolutionFilePath = null;
```

Comment:
After a bracket, should put a new line break.

```suggestion

			SolutionFilePath = null;
```